pyproject.toml

```diff
@@ -34,6 +34,8 @@ cdflib = "^1.3.1"
 psycopg = {extras = ["binary"], version = "^3.2.1"}
 prefect = {extras = ["shell"], version = "^3.1.7"}
 prefect-docker = "^0.6.1"
+psycopg2 = "^2.9.10"
+psycopg2-binary = "^2.9.10"
```

Comment:
I dont think this should be necessary - we are supposed to be using v3 of the driver. https://www.psycopg.org/psycopg3/docs/basic/install.html
Suspect we went wrong with the connection string or something